code/src/App.js

```diff
@@ -1,9 +1,130 @@
-import React from 'react'
+import React, { useState } from 'react'
+import './App.css'
+
+
+
+const ageGroups = ["young", "old", "perfect"]
 
 export const App = () => {
+  const [ageGroup, setAgeGroup] = useState("perfect")
+  const [animal, setAnimal] = useState("")
+  const [showResult, setShowResult] = useState(false)
+  const [bestFriend, setBestFriend] = useState("")
+  const [fun, setFun] = useState("fun")
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    setShowResult(true)
+  }
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <div className="result">
+      {showResult && (
+        <div>
+          <h2>So, I think I know you now:</h2>
+          <p> You are a {fun === "fun" ? "fun" : "no fun"} {ageGroup} {animal} and your best friend is {bestFriend}.</p>
+        </div>
+
+      )}
+
+
+
+      {!showResult && (
+        <form onSubmit={handleSubmit}>
+
+          <section className="questions">
+            <h1>Can I ask you a few questions?</h1>
+            <div className="ages">
+              <h3>How old are you?</h3>
+            </div>
+            <div>
+              {ageGroups.map(group => (
+                <label key={group}>
+                  <input
+                    type="radio"
+                    value={group}
+                    onChange={event => setAgeGroup(event.target.value)}
+                    checked={ageGroup === group}
+                    required
+                  />
+                  <span className="fakeRadio" />
+                  {group}
+                </label>
+              ))}
+
+            </div>
+
+            <div className="spiritAnimal">
+
+              <h3>What is your spirit animal?</h3>
+            </div>
+            <label>
+              <div>
+                <input className="inputField" type="text"
+                  value={animal}
+                  onChange={(event) => setAnimal(event.target.value)} required />
```

Comment:
Write
```
<input 
   className="inputField" 
   type="text"
   value={animal}
   onChange={(event) => setAnimal(event.target.value)} 
   required 
/>
```
Or 
```
<input className="inputField" type="text" value={animal} onChange={(event) => setAnimal(event.target.value)}  required />
```
One of them instead of both